engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-packages-modal.controller.js

```diff
@@ -134,10 +135,14 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkPackagesModa
                 $scope.packageActionFormValues.remoteAction = 'package_' + action;
             }
 
-            if (selectedHosts.included.ids) {
+            if (selectedHosts.included.ids.length > 0) {
                 $scope.packageActionFormValues.hostIds = selectedHosts.included.ids.join(',');
             }
 
+            if ($scope.allHostsSelected) {
```

Comment:
oh, one thing you'll want to check on (sorry, i just realized)

The way the code is written will send both the ids and the scoped search value. From what I can tell, I think we only care about one or the other. Let me know if you get a different impression from the UI. If this is true, then we should only send one or the other:

```
if ($scope.allHostsSelected) {
 ...
} elseif (selectedHosts.included.ids.length > 0) {
...
}
```

Reply:
Based on the testing I did, once the scoped search is populated, the host ids didn't seem to make any difference if present.

In any case, seems like this would reduce confusion in the future.